Proton/Sources/Core/ListParser.swift

```diff
@@ -132,7 +132,7 @@ public struct ListParser {
         var startIndex = 0
 
         for newlineRange in newlineRanges {
-            let isNewlineSkipMarker = text.attributedSubstring(from: newlineRange).attribute(.skipNextListMarker, at: 0, effectiveRange: nil) != nil
+            let isNewlineSkipMarker = text.attribute(.skipNextListMarker, at: newlineRange.location, effectiveRange: nil) != nil
```

Comment:
This is a nice example of what I'm trying to avoid: Allocating a new `NSAttributedString` for _every_ newline only to check the existence of an attribute at a known location.